corehq/apps/receiverwrapper/repeater_generators.py

```diff
@@ -37,6 +37,13 @@ def get_payload(self, repeat_record, payload_doc):
         return payload_doc.to_xml(self.repeater.version or V2, include_case_on_closed=True)
 
 
+@RegisterGenerator(CaseRepeater, 'case_json', 'JSON', is_default=False)
+class CaseRepeaterJsonPayloadGenerator(BasePayloadGenerator):
+    def get_payload(self, repeat_record, payload_doc):
+        del payload_doc['actions']
+        return json.dumps(payload_doc.get_json())
```

Comment:
Sorry, being picky but I think you should do `get_json(lite=True)` to avoid having to fetch the reverse case indices.